sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/SimpleAccessorNameNotationCheck.java

```diff
@@ -298,8 +294,7 @@ private static boolean containsOnlyReturn(DetailAST methodBody) {
     private static boolean isCorrectReturn(DetailAST literalReturn) {
         //two child: EXPR and SEMI
         return literalReturn.getChildCount() == 2
-                && literalReturn.getFirstChild().getType() == TokenTypes.EXPR
-                && literalReturn.getLastChild().getType() == TokenTypes.SEMI;
```

Comment:
return with expression is always guaranteed a semicolon, right? we already verify child count is 2.